tips-and-tricks.md

```diff
@@ -1,6 +1,56 @@
 # Tips and Tricks
 
-## Managing your data and the storage spaces
+## Python
+
+### Install miniconda
+
+This is the simplest approach if you are familiar with `conda`.
+
+To install `miniconda` in `$WORK/miniconda3`:
+```sh
+# download Miniconda installer
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh
+# install Miniconda
+MINICONDA_PATH=$WORK/miniconda3
+chmod +x miniconda.sh && ./miniconda.sh -b -p $MINICONDA_PATH
+# make sure conda is up-to-date
+source $MINICONDA_PATH/etc/profile.d/conda.sh
+conda update --yes conda
+```
+
+You probably want to add this to your `.bashrc` to activate your conda environment on each login:
```

Comment:
The suggested method (that works for people using other shells than Bash, but also for Bash) is to type `conda init`.

Reply:
Good point: I think `conda init` adds the necessary lines in `.bashrc` I'll tweak that.

Reply:
Ok!